test/e2e/framework/addon/vault/setup.go

```diff
@@ -208,7 +232,7 @@ func (v *VaultInitializer) Init() error {
 			return fmt.Errorf("error parsing proxy URL: %s", err.Error())
 		}
 		var lastError error
-		err = wait.PollUntilContextTimeout(context.TODO(), time.Second, 20*time.Second, true, func(ctx context.Context) (bool, error) {
+		err = wait.PollUntilContextTimeout(context.TODO(), time.Second, 45*time.Second, true, func(ctx context.Context) (bool, error) {
```

Comment:
This is to avoid a race condition while the Vault e2e addon is starting, [here](https://github.com/cert-manager/cert-manager/blob/b120669e8243b6ab868375495bbfc1e77d722be0/test/e2e/framework/addon/globals.go#L69) a new Vault addon with mTLS is synchronously started, however the creation of Vault client and the port-forward in the e2e framework was already dispatched asynchronously, the previous 20 seconds was a valid timeout when only one Vault addon was being created, since there is two now, that timeout has to be increased, otherwise the port-forward would fail because the second Vault addon was not yet ready.